client/scraper/metadata.py

```diff
@@ -166,14 +172,14 @@ def generic_parse_content(self, response):
 
             self.prop_map[FieldKeyword.DESC] = self.get_desc(response)
 
-            self.prop_map[FieldKeyword.FAVICON] = self.get_favicon_url(response)
-
-            self.prop_map[FieldKeyword.IMAGES] = self.get_images_list(response)
+            self.prop_map[FieldKeyword.IMAGES] = self.get_images_list(response, self.prop_map[FieldKeyword.PROVIDER_URL])
 
```

Comment:
Just so we keep this consistent, just retrieve the `provider_url` from `get_images_list` rather than passing it in. Both functions have access to `self`